app/models/miq_widget.rb

```diff
@@ -532,6 +537,23 @@ def sync_schedule(schedule_info)
     sched
   end
 
+  def existing_scheduler
```

Comment:
Should this be `existing_schedule` without the `r`?

Reply:
ok